servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractRest.java

```diff
@@ -55,26 +50,17 @@ public abstract class AbstractRest {
     Locale.setDefault(Locale.ENGLISH);
   }
 
-  protected void init(URI uri) {
-    NessieApiV1 api = HttpClientBuilder.builder().withUri(uri).build(NessieApiV1.class);
-
-    ObjectMapper mapper =
-        new ObjectMapper()
-            .enable(SerializationFeature.INDENT_OUTPUT)
-            .disable(SerializationFeature.FAIL_ON_EMPTY_BEANS);
-    HttpClient.Builder httpClient = HttpClient.builder().setBaseUri(uri).setObjectMapper(mapper);
-    httpClient.addResponseFilter(new NessieHttpResponseFilter(mapper));
-
-    init(api, httpClient);
+  protected void initApi(URI nessieApiUri) {
+    NessieApiV1 api = HttpClientBuilder.builder().withUri(nessieApiUri).build(NessieApiV1.class);
+    initApi(api);
   }
 
-  protected void init(NessieApiV1 api, @Nullable HttpClient.Builder httpClient) {
+  protected void initApi(NessieApiV1 api) {
     this.api = api;
   }
 
-  @BeforeEach
-  public void setUp() {
-    init(URI.create("http://localhost:19121/api/v1"));
```

Comment:
this hardcoding of 19121 is quarkus specific even though the `jax-rs-tests` module shouldnt be...
all tests from `quarkus-server` depended on this hardcoding in the base class, while `AbstractTestJerseyRest` avoided this hard-coding by overriding `setUp`.